python/foglamp/plugins/south/coap_listen/coap_listen.py

```diff
@@ -193,44 +199,43 @@ class CoAPIngest(aiocoap.resource.Resource):
         # https://docs.google.com/document/d/1rJXlOqCGomPKEKx2ReoofZTXQt9dtDiW_BHU7FYsj-k/edit#
         # and will be moved to a .rst file
 
-        code = aiocoap.numbers.codes.Code.INTERNAL_SERVER_ERROR
-        increment_discarded_counter = True
+        code = aiocoap.numbers.codes.Code.VALID
+        # TODO: Decide upon the correct format of message
         message = ''
-
         try:
             if not Ingest.is_available():
+                Ingest.increment_discarded_readings()
                 message = '{"busy": true}'
-            else:
-                payload = cbor2.loads(request.payload)
-
-                if not isinstance(payload, dict):
-                    raise ValueError('Payload must be a dictionary')
+                raise aiocoap.error.CommunicationKilled(message)
```

Comment:
for reference https://github.com/chrysn/aiocoap/blob/master/aiocoap/error.py#L176